packages/orbit-components/src/Badge/Badge.ct-story.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import Airplane from "../icons/Airplane";
+import { TYPE_OPTIONS } from "./consts";
+
+import Badge from ".";
+
+export default function BadgeVisualTest() {
+  return (
+    <div className="space-x-xs flex flex-col">
```

Comment:
Is the `flex-col` expected?

Reply:
@mvidalgarcia yes, expected, I changed them in the most cases to be in the column, as they sometimes are overflowed and not fit into viewport

Reply:
I would add only change this screenshot by adding spacings and not to be such wide :)